snap/info.go

```diff
@@ -636,7 +639,8 @@ type AppInfo struct {
 	// TODO: this should go away once we have more plumbing and can change
 	// things vs refactor
 	// https://github.com/snapcore/snapd/pull/794#discussion_r58688496
-	BusName string
+	BusName  string
+	CommonID string
```

Comment:
I think it should go into the Name field grouping

Reply:
Moved